INSTALL.md

```diff
@@ -1,119 +1,181 @@
-Installation instructions for Frescobaldi
-=========================================
+# Installation instructions for Frescobaldi
 
-You can run Frescobaldi without installing. Just unpack and then run:
 
-    python3 frescobaldi
+## Using a pre-built package
 
-See below for the required dependencies.
+Each release comes with installers for Windows and macOS.  These are available
+from
 
-The Frescobaldi package is based on distutils. No build process is needed as
-Frescobaldi is fully written in the interpreted Python language. To install in
-the default locations:
+https://github.com/frescobaldi/frescobaldi/releases/
 
-    sudo python3 setup.py install    # install to /usr/local
-    python3 setup.py install --user  # install to ~/.local
+This is the simplest way of installing Frescobaldi under Windows or macOS.
 
-To run Frescobaldi, then simply type
+Under Linux, your distribution probably provides a `frescobaldi` package.  If
+so, installing it is the easiest route, even though it might be out of date.
+Run `sudo apt install frescobaldi` on Debian/Ubuntu or `sudo dnf install
+frescobaldi` on Fedora.
 
-    frescobaldi
+You can also install Frescobaldi from Flathub using Flatpak to get a more
+up-to-date application.
 
-If you want to install into /usr instead of /usr/local:
+https://flathub.org/apps/org.frescobaldi.Frescobaldi
 
-    sudo python3 setup.py install --prefix=/usr
+**In this case, be sure to grant the Frescobaldi application access on your home
+directory in the Flatseal application, or you will not be able to use LilyPond
+files other than in specific directories (like `~/Documents`) due to the
+sandboxing that Flatpak imposes.**
 
-If you have a Debian-based system such as Ubuntu, and you get the error
-message "ImportError: No module named frescobaldi_app.main", try:
+Also, if you previously installed Frescobaldi via your distribution, remember to
+remove it before installing it through Flatpak.
 
```

Comment:
I know you wrote this to make things simple for the user. It's good that distro package installation and flatpak installation can coexist, but it's unfortunate that currently you see only one application in the desktop environment. Maybe we can tell the full truth to let the user make his/her choice. For example:

> "If you previously installed Frescobaldi via your distribution, be aware that it can coexist with the Flatpak installation but your desktop environment might launch only the distro package. You have two options: 1- open the flatpak app from a terminal with the command `flatpak run org.frescobaldi.Frescobaldi`; 2- remove the distro package. "

Writing complete and concise doc is difficult. I don't like my wording, but you see what I'm trying to communicate.